Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```
@@ -49,6 +51,10 @@
         whitelist:
           tags:
           - Powerdrill
+      rpd:
+        whitelist:
+          tags:
+          - RPD
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)